.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
             - run:
                 name: Test WordPress
                 # @todo, change to pantheon-systems
-                command: ./.circleci/test-example-repo.sh "pantheon-systems/example-wordpress-composer:dev-master"
+                command: ./.circleci/test-example-repo.sh "pantheon-systems/example-wordpress-composer:dev-dynamic-build-tools-version"
```

Comment:
n.b. `dev-test-2.x` in the example-drops-8-composer version (per my comment https://github.com/pantheon-systems/example-wordpress-composer/pull/15#issuecomment-316473855)